front/src/config/i18n/en.json

```diff
@@ -791,6 +791,18 @@
         "saveLabel": "Save configuration"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connect now",
+      "apiKeyLabel": "Then, enter your client_id and secret_id key here:",
```

Comment:
I'm not sure secret_id is correct, I think we talk about client id and client secret.

Reply:
i fix it